### PR TITLE
build: add readme to nuget packages

### PR DESCRIPTION
**Pull request by philasmar**
*Description of changes:*
Add readme to nuget packages

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <PackageProjectUrl>https://github.com/aws/aws-dotnet-deploy</PackageProjectUrl>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\public.snk</AssemblyOriginatorKeyFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```
Comment (normj) [inline src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj:14]: Seeing that we don't want/expect users to use this package I don't think we should reuse the main README file. Either have a separate README file how we don't expect you to use this or just don't have one.

**Review by normj — CHANGES_REQUESTED** [review]

File: src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj (hunk 1 above)
Comment (philasmar) [reply src/AWS.Deploy.ServerMode.Client/AWS.Deploy.ServerMode.Client.csproj]: Removed it.

File: src/AWS.Deploy.Recipes.CDK.Common/AWS.Deploy.Recipes.CDK.Common.csproj [hunk 2]
```diff
@@ -24,6 +25,7 @@
 
   <ItemGroup>
     <None Include="..\..\icon.png" Pack="true" PackagePath="" />
+    <None Include="..\..\README.md" Pack="true" PackagePath="" />
```
Comment (normj) [inline src/AWS.Deploy.Recipes.CDK.Common/AWS.Deploy.Recipes.CDK.Common.csproj:28]: Can we put in a very small separate README file for `AWS.Deploy.Recipes.CDK.Common` that just says this a packages used for .NET deployment project created by `AWS.Deploy.Tool`?  I think reusing the README for AWS.Deploy.Tool will be confusing.

**Review by normj — CHANGES_REQUESTED** [review]
Sorry I just have mentioned this one package as well in my last review.

File: src/AWS.Deploy.Recipes.CDK.Common/AWS.Deploy.Recipes.CDK.Common.csproj (hunk 2 above)
Comment (philasmar) [reply src/AWS.Deploy.Recipes.CDK.Common/AWS.Deploy.Recipes.CDK.Common.csproj]: Done

**Review by normj — APPROVED** [review]

**Pull request merged**